python/mathsys/main/parser.py

```
@@ -1,122 +1,122 @@
+    factors: list[Factor]
+
+
+#
+#   5ºLEVEL
+#
+
+# 5ºLEVEL -> FACTOR
+@dataclass
+class Factor:
+    type: [
+        "unsigned" | "signed", 
```

Ruff (0.12.2)

Undefined name `unsigned`

(F821)